dids/src/main/kotlin/web5/sdk/dids/Json.kt

```diff
@@ -0,0 +1,95 @@
+package web5.sdk.dids
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.core.JsonGenerator
+import com.fasterxml.jackson.core.JsonParser
+import com.fasterxml.jackson.core.JsonProcessingException
+import com.fasterxml.jackson.databind.DeserializationContext
+import com.fasterxml.jackson.databind.JsonDeserializer
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.JsonSerializer
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.ObjectWriter
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.databind.SerializerProvider
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+import com.nimbusds.jose.jwk.JWK
+import web5.sdk.dids.didcore.Purpose
+import java.io.IOException
+
+/**
+ * A singleton for json serialization/deserialization, shared across the SDK as ObjectMapper instantiation
+ * is an expensive operation.
+ * - Serialize ([stringify])
+ *
+ * ### Example Usage:
+ * ```kotlin
+ * val offering = Json.objectMapper.readValue<Offering>(payload)
+ *
+ * val jsonString = Json.stringify(myObject)
+ *
+ * val node = Json.parse(payload)
+ * ```
+ */
+public object Json {
```

Comment:
Optional nit: Move this object into the `common` package

Reply:
done!